core/src/main/java/org/folio/tlib/postgres/cqlfield/PgCqlFieldText.java

```diff
@@ -92,18 +106,28 @@ public String handleTermNode(CQLTermNode termNode) {
       return s;
     }
     String base = termNode.getRelation().getBase();
+    if (enableLike && "=".equals(base)) {
+      // not including "<>" as it is exact match, just like ==
+      StringBuilder cqlTerm = new StringBuilder();
+      if (cqlTermToPgTermLike(termNode, cqlTerm)) {
+        return column + " LIKE '" + cqlTerm + "'";
+      }
+    }
     boolean fulltext = language != null;
     if (fulltext) {
-      // only for relations "=", "all" is full-text search performed
-      fulltext = "=".equals(base) || "all".equals(base);
-    }
-    if (!fulltext) {
-      return column + " " + handleUnorderedRelation(termNode)
-          + " '" +  cqlTermToPgTermExact(termNode) + "'";
+      String func = null;
+      if ("adj".equals(base) || "=".equals(base)) {
+        func = "phraseto_tsquery";
+      } else if ("all".equals(base)) {
+        func = "plainto_tsquery";
+      }
+      if (func != null) {
```

Comment:
I don't think withFullText should disable exact match (which is the default). Now that relation does distinguish the cases as is done for RMB and in according to the spec, I think it's good to have a way to disable exact matches.. If that was to be used for this without full-text the only functionality would be IS NULL / IS NO NULL. I'm open to other ideas .. eg something that is closer to what is defined indices.. Eg we use more PostgresQL terminology and just say what indieces are "present" and then it will adjust accordingly.

Reply:
How about this? Leave all disabled by default. One has to enable exact searches with withExact().

I like that better than having to "disable" things.

So the only thing (enabled by default) is IS NULL, IS NOT NULL.

Reply:
Ok. Thx for checking